github/ci/prow-deploy/kustom/base/manifests/local/prow-test-subset-deployment.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: test-subset
+  labels:
+    app: test-subset
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: prow-test-subset
```

Comment:
here to keep preifx prow ?
(otoh on line 4 and 6 it is without)

Reply:
IMHO remove it also here